src/SignalRServiceExtension/NegotiationContextAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    /// <summary>
+    /// An input binding attribute to bind <see cref="NegotiationContext"/> to the function parameter.
+    /// </summary>
+    /// <remarks>Designed for function languages except C# to customize negotiation routing.</remarks>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    [Binding]
+    public class SignalRNegotiationAttribute : NegotiationBaseAttribute
+    {
```

Comment:
What do you think about `SignalRNegotiationAttribute` instead of `SignalRNegotiationContext`? As `HttpTrigger` doesn't use `HttpResponse` as the attribute name, so I think use `SignalRNegotiation` would be more concise.

Reply:
Looks good to me